GetDisplayUserInfo/GetDisplayUserInfo/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace GetDisplayUserInfo
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("What is your first name?");
+            string firstName = Console.ReadLine();
+
+            Console.WriteLine("Hello, " + firstName + ". " + "It's very nice to meet you!");
+
+            Console.WriteLine("What is your last name?");
+            string lastName = Console.ReadLine();
```

Comment:
This pair of lines (14-15) looks like 9-10, 19-20, 26-27, and 34-35. Could this perhaps be put into a method and called when needed? Something like:
```CS
static string GetAnswer(string question)
{
  Console.WriteLine(question);
  return Console.ReadLine();
}
```

Then, on line 9 for example, `string firstName = GetAnswer("What is your first name?");`